FirebaseAuth/Sources/Backend/FIRIdentityToolkitRequest.m

```diff
@@ -82,6 +82,10 @@ - (BOOL)containsPostBody {
   return YES;
 }
 
+- (nullable NSString *)queryParams {
+  return nil;
```

Comment:
Should there be a TODO here?

Reply:
This is intentional. This method returns `nil` by default in the base class and it's up to its subclass to override and return the params.